package.json

```diff
@@ -42,12 +42,16 @@
     "url": "https://github.com/xFAANG/askql/issues"
   },
   "homepage": "https://askql.org/",
-  "dependencies": {},
+  "dependencies": {
+    "cors": "^2.8.5",
+    "nanoid": "^3.1.10"
+  },
```

Comment:
These are all dev-dependencies for now. When we add package-level package.json file, then we can do this differentiation. This means that the playground is only available for development of this project.

Reply:
OK, done.